### ISSUE TITLE
Warn user of dangerous Find in files queries

### ISSUE THREAD

**Issue by elsassph** [fdorg/flashdevelop#713]
Like using a 1 char common letter search.

**Comment by larsiusprime** [comment]
YES!

**Comment by Gama11** [comment]
Would be nice if searches with tons of results don't cause FD to hang as well, I've managed to crash it a couple of times this way.

**Comment by elsassph** [comment]
Yeah a security when reaching too many results would be sensible...
